server/views/pages/signin.html

```diff
@@ -37,15 +37,15 @@
   {{ topBar({
```

Comment:
don't think this page is used! Can probably be deleted

Reply:
Good spot @andrewrlee This is an old HTML template that was part of the old sign in designs. I'll raise a ticket to remove this code.